man/pcr-oracle.8.in

```diff
@@ -495,6 +495,18 @@ events (tracked in PCR9). In this case, it would be totally sufficient
 to stop processing after grub loaded \fPgrub.cfg\fP from the EFI
 System Partition.
 .TP
+.BI --next-kernel " id
```

Comment:
That is too generic for my taste, and it's also not accurate. The new option doesn't add anything, it influences the PCR prediction. Without the option, the PCR "prediction" will use the kernel/initrd/option hashes from the original UEFI event, and with the option, it will use the specified boot entry.

So maybe --boot-entry is a better name

Reply:
`--boot-entry` sounds good too.

> The new option doesn't add anything

I think it does: it adds a new json section in the file specified in the output. Am I misreading [sdb_policy_file_add_entry](https://github.com/okirch/pcr-oracle/commit/f0c0d27630c1c7baea5f00a762975a1ec08a2b95#diff-d8f84cf82712a3ff1d2b4b08dadfbc2dcafd61c81a6c351729b271b00ff299edR497) in `main`?

Reply:
No. Signing a policy adds a new section:
 pcr-oracle --target-platform systemd .. sign
The new option just modifies which kernel and initrd will be hashed into the PCRs, so it's something like this:

for entry in /boot/efi/loader/entries/*.conf; do
    id=$(basename "$entry" .conf)
    pcr-oracle --target-platform systemd --boot-entry "$id" ... sign
done